src/main/java/ch/njol/skript/expressions/ExprAppliedEffect.java

```diff
@@ -0,0 +1,66 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import com.destroystokyo.paper.event.block.BeaconEffectEvent;
+import org.bukkit.event.Event;
+import org.bukkit.potion.PotionEffectType;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Applied Effect")
```

Comment:
```suggestion
@Name("Applied Beacon Effect")
```
applied effect is too broad imo